getx_demo/lib/models/account.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:get/get.dart';
+import 'package:uuid/uuid.dart';
+
+import 'package:getx_demo/models/transaction.dart';
+
+enum LoanDecisionRules {
+  uninitiated,
+  approved,
+  declined,
+  waiting,
+}
+
+class Account extends GetxController {
```

Comment:
This should not be a `GetxController`.

Lets use `freezed` package to generate models.